### ISSUE TITLE
Meteor browser package is built to server folder.

### ISSUE THREAD

**Issue by vardhans** [meteor/meteor#6797]
I am using meter version 1.1.6
When I start my project with cmd `meteor --setting settings.json`
I am getting below error.

```
W20160412-23:25:45.062(10)? (STDERR)
W20160412-23:25:45.062(10)? (STDERR) /Users/vardhan/.meteor/packages/meteor-tool/.1.1.3.bgnjgr++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:245
W20160412-23:25:45.063(10)? (STDERR) throw(ex);
W20160412-23:25:45.063(10)? (STDERR) ^
W20160412-23:25:45.264(10)? (STDERR) TypeError: Cannot read property 'document' of undefined
W20160412-23:25:45.264(10)? (STDERR) at saveAs (packages/pfafman_filesaver.js:38:15)
W20160412-23:25:45.264(10)? (STDERR) at Package (packages/pfafman_filesaver.js:281:2)
W20160412-23:25:45.264(10)? (STDERR) at packages/pfafman_filesaver.js:302:4
W20160412-23:25:45.264(10)? (STDERR) at packages/pfafman_filesaver.js:311:3
W20160412-23:25:45.264(10)? (STDERR) at /Users/vardhan/Documents/matrix/test/project-aegis/.meteor/local/build/programs/server/boot.js:222:10
W20160412-23:25:45.264(10)? (STDERR) at Array.forEach (native)
W20160412-23:25:45.264(10)? (STDERR) at Function..each..forEach (/Users/vardhan/.meteor/packages/meteor-tool/.1.1.3.bgnjgr++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160412-23:25:45.265(10)? (STDERR) at /Users/vardhan/Documents/matrix/test/project-aegis/.meteor/local/build/programs/server/boot.js:117:5
```

up on investigation i found that `pfafman:filesaver` meteor package is build to server (`project_folder/.meteor/local/build/programs/server/packages`) where its supposed to be on web.browser(`project-aegis/.meteor/local/build/programs/web.browser/packages`)

It was building fine before not sure what happened now. Can anyone help me with this.?

**Comment by zhaoyao91** [comment]
meteor 1.2.1, seems same problem
https://forums.meteor.com/t/published-package-mix-all-files/21305/1

**Comment by jkchong9** [comment]
It seems like coming from new version of pfafman:filesaver.
Maybe compatibility issue between new packages and old meteor version?
There is no issue with version 0.2.2.  
You can go back to 0.2.2 by: 
c:> meteor add pfafman:filesaver@=0.2.2

**Comment by zhaoyao91** [comment]
@jin-chong I think this is not a problem of a single package. I have wrote packages built and work fine when installed locally, but after I publish them to atmosphere, all server and client code are mixed

**Comment by laosb** [comment]
According to @zhaoyao91 , This bug has been fixed in 1.3. Comment if you still suffer.
